Update che4z-installing.adoc (#884)
codenvy/che-docs: src/main/pages/che-7/extensions/che4z-installing.adoc

:context: installing-che4z

> To download and install the Eclipse Che4z plugins, follow the instructions on link:https://projects.eclipse.org/projects/ecd.che.che4z/downloads[this page]..Prerequisites

Reviewer: ```suggestion
To download and install the Eclipse Che4z plugins, follow the instructions at the link:https://projects.eclipse.org/projects/ecd.che.che4z/downloads[Eclipse Che4z page].

.Prerequisites
```

- Minimum of 2 CPUs and 4GB RAM
- Docker
- Internet access

.Procedure

. Install Eclipse Che by running the following command. Replace `__<DATA_FOLDER>__` with the location of user data on the host machine:
+
[subs="+quotes"]
----
$ docker run -it --rm -e CHE_MULTIUSER=true -e CHE_HOST=`hostname -f` -e \
  CHE_WORKSPACE_VOLUME=/var/run/docker.sock:/var/run/docker.sock -v \
  /var/run/docker.sock:/var/run/docker.sock -v /__<DATA_FOLDER>__:/data eclipse/che:7.0.0-beta-4.0 start
----
+

> For more information on configuring Eclipse Che with Docker, see https://www.eclipse.org/che/docs/che-6/docker-multi-user.html[Installation on Docker].

Reviewer: Che 6 docs have now been removed, so we shouldn't link to them.

Author: Is the content covering this point (configuring eclipse che with docker) now covered elsewhere or is it now deprecated?

Reviewer 2: We do not support configuring Che with Docker anymore. You can install Che locally using minikube, minishift, microk8s or docker4mac/docker4windows kubernetes.

. Log in to Che using the default administrator account (username = `admin`, password = `admin`).

. (Optional) Configure the Keycloak security server by following the instructions https://www.eclipse.org/che/docs/che-6/user-management.html#che-and-keycloak[Managing users].

. Download and unzip the installer.

. Run `start_installer.sh` (on Linux) or `start_installer.cmd` (on Windows).

. When prompted, enter the Che URL and administrator account credentials.

== Creating a workspace

. Log in to Che. Use your credentials or the default administrator account (username = `admin`, password = `admin`).

. Select *Workspaces -> Add Workspace*.

. In *Select Stack*, select *view all*, then choose the *Broadcom Mainframe* stack.

. Create and open the workspace.

> You created a workspace and can now use the Eclipse Che4z extensions.

Reviewer: ```suggestion
A workspace has now been created with Eclipse Che4z extensions available.
```
